### ISSUE TITLE
On small screens, characters-per-line in cards are too few. Optimum range on all screen sizes should be ~50-60

### ISSUE THREAD

**Issue by ErnieAtLYD** [wherebyus/general-tasks#1153]
Characters per line
TEG 4/12 web full-width: 54, 59, 58, 59, 52
TEG 4/12 web iPhone 7 width: 34, 33, 27, 35, 34

References
Optimal readability is considered to be [50-60 characters per line](https://baymard.com/blog/line-length-readability?utm_source=designernews).

Feedback
https://twitter.com/clstokes/status/1116765792215814144?s=12

**Comment by michaelschofield** [comment]
@ErnieAtLYD @carogea This isn't really a _bug_ insomuch as a preference. Before any work happens on this ticket, I think we should chat briefly together to talk about *how do we approach these design tweaks / prototypes / etc.* This was spawned from a single twitter comment, right? Which isn't identifying a _bug_, but a preference.

My suggestion is that for things like this:

- [ ] The problem needs to be sufficiently validated. Questions to answer: what does "sufficiently validated" mean?
- [ ] The solution, if a design solution, needs to be prototyped and not merged into develop etc. Especially when ti comes to card padding, that will have cascading implications on everything used cards - for instance. So a "card" related fix for just newsletters doesn't fly, as it impacts cards elsewhere. We also don't want to get in the habit of newsletter-specific cards as that's not scalable. Questions to answer: What's the workflow for this prototype? Who needs to sign off? etc.

**Comment by michaelschofield** [comment]
I created #1159

**Comment by michaelschofield** [comment]
![image.png](https://images.zenhubusercontent.com/5907891d51aa2c30577cf4d5/0b94efe9-1feb-4e6b-83b7-fee7ca915043)

**Comment by michaelschofield** [comment]
Thumbs-up https://wherebyus.slack.com/archives/C61F5AE5R/p1555686946003200
